frontend/elements/src/entry/admin/locale/landing.ts

```diff
@@ -0,0 +1,206 @@
+                grid-column: 4;
+            }
+
+            .table {
+                grid-column: 1 / -1;
+                display: grid;
+                grid-template-columns: repeat(11, auto);
+            }
+            ::slotted([slot=rows]) {
+                display: contents;
+            }
+            :host([sortOrder=asc]) {
+                --sort-arrow: '⇩';
+            }
+            :host([sortOrder=desc]) {
+                --sort-arrow: '⇧';
+            }
+
+            dialog {
+                border-radius: 8px;
+                border: solid #00000040 1px;
+            }
+            dialog::backdrop {
+                background-color: #00000060;
+            }
+            .column-selection-contents {
+                display: grid;
+                grid-template-columns: repeat(2, 200px);
+            }
+            .column-selection-contents hr {
+                grid-column: 1 / -1;
+                width: 100%;
+            }
+            .column-selection-contents header {
+                grid-column: 1 / -1;
+                text-align: center;
+            }
+            .column-selection-contents ul {
+                list-style: none;
+                padding: 0;
+                margin: 0;
+            }
+            .column-selection-contents .actions {
+                grid-column: 1 / -1;
+                display: flex;
+                justify-content: flex-end;
+                column-gap: 10px;
+            }
+        `];
+    }
+
+    @property({type: Array})
+    public bundles: Bundle[] = [];
+
+    @property({type: Array})
+    public columns: Column[] = []
+
+    @property({type: Boolean, reflect: true})
+    public saving = false;
+
+    @property({reflect: true})
+    public sortOrder: 'asc' | 'desc' = 'asc';
+
+    @query('dialog', true)
+    public dialog!: HTMLDialogElement;
+
+    private onBundleSelect(e: Event) {
+        const select = e.target as HTMLSelectElement;
+        const options = select.options;
+        this.bundles.forEach((b, i) => {
+            b[1] = options[i].selected;
+        });
+        
+        this.dispatchEvent(
+            new CustomEvent("selected-bundle-change", {
+                detail: this.bundles
+            })
+        );
+    }
+
+    private addEntry() {
+        this.dispatchEvent(
+            new CustomEvent("add-entry")
+        );
+    }
+
+    private showSelectColumns() {
+        // this.dialog.showModal();
+        // cant get @query to work, dono why
+        this.renderRoot.querySelector("dialog")!.showModal();
+    }
+
+    private dismissSelectColumns() {
+        // this.dialog.showModal();
+        // cant get @query to work, dono why
+        this.renderRoot.querySelector("dialog")!.close();
+    }
+
+    public render() {
+        return html`
+            <div class="saving-indicator"></div>
+            <select multiple @change="${this.onBundleSelect}">
+                ${this.bundles.map(([bundleName, selected]) => {
```

Comment:
would be better to slot these in so that you can handle the events on the Rust side, instead of relying on the CustomEvent